styles/styles.css

```diff
@@ -255,3 +255,7 @@ main .section.highlight {
   margin: 0;
   padding: 40px 0;
 }
+.sectioncolor{
```

Comment:
Where are you using it?

Reply:
Always use hyphen separator and don't mention section keyword

Reply:
I remove the section keyword.
And this is about day (1 part-2) Assignment 3